artifactory/cli.go

```diff
@@ -662,6 +667,58 @@ func GetCommands() []cli.Command {
 				return repoDeleteCmd(c)
 			},
 		},
+		{
+			Name:         "replication-template-create",
+			Aliases:      []string{"rtc"},
+			Flags:        getTemplateUsersFlags(),
+			Usage:        replicationtemplate.Description,
+			HelpName:     common.CreateUsage("rt rtc", replicationtemplate.Description, replicationtemplate.Usage),
+			UsageText:    replicationtemplate.Arguments,
+			ArgsUsage:    common.CreateEnvVars(),
+			BashComplete: common.CreateBashCompletionFunc(),
+			Action: func(c *cli.Context) error {
+				return replicationTemplateCmd(c)
+			},
+		},
+		{
+			Name:         "replication-job-create",
```

Comment:
replication-job-create --> replication-create
(let's change the short name also and the help.go file)